admin/load.php

```diff
@@ -340,8 +363,10 @@ function perflab_get_module_data( $module_file ) {
 	// Translate fields using low-level function since they come from PHP comments, including the necessary context for
 	// `_x()`. This must match how these are translated in the generated `/module-i18n.php` file.
 	$translatable_fields = array(
-		'name'        => 'module name',
-		'description' => 'module description',
+		'name'            => 'module name',
+		'description'     => 'module description',
+		'enabled-notice'  => 'module enabled notice',
+		'disabled-notice' => 'module disabled notice',
```

Comment:
See above, can we revert this in favor of a hard-coded message in `admin/load.php`? This way we can explore a proper API for this separately without having that conversation block this PR.

Reply:
Done